specification/electronic-prescription-service-api.yaml

```diff
@@ -55,19 +55,31 @@ info:
     - [Personal Demographics Service - FHIR API](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir) - use this to obtain patient data
 
     ## API status and roadmap
-    The immediate EPS development priority is to support the rollout of the next-gen FHIR APIs for primary care and secondary care prescribing and dispensing. Alongside this we will be working on integrating a new FHIR-compliant prescription tracker into the next gen APIs.
+    The immediate priority of EPS development is to support the rollout of the next-gen FHIR APIs for primary care and secondary care prescribing and dispensing. Alongside this we will be working on integrating a new FHIR-compliant prescription tracker into the next gen APIs.
     
-    To suggest, comment or vote on features for this API, see our [interactive product backlog](https://nhs-digital-api-management.featureupvote.com/?tag=eps-fhir-api).
-
-    If you have any other queries, please [contact us](https://digital.nhs.uk/developer/help-and-support).
-
     ### Prescribing API
-    Prescribing flows for EPS FHIR API are now in a state of technical go-live awaiting our first of type integration.
-
-    The API is currently available for sandbox testing and integration testing, but not for production use.
-
+    There are three types of medication workflows
+    
+    #### Primary care medication workflow
+    In primary care medication workflow, the prescription is sent from a primary care service, such as a GP practise to a Pharmacy. For this workflow, the API is in [alpha](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#api-status), meaning:
```

Comment:
Couple of minor typos:
_In primary care medication workflow_ > _In a primary care medication workflow_
_GP practise_ > _GP practice_

Reply:
Corrected the typos